implementations/rust/ockam/ockam_vault/src/secret_impl.rs

```diff
@@ -39,6 +48,25 @@ impl Vault {
                 ))
                 .await?
             }
+            SecretType::NistP256 => '_block: {
+                #[cfg(feature = "aws")]
+                if attributes.persistence() == SecretPersistence::Persistent {
+                    if let Some(kms) = &self.aws_kms {
+                        if let Secret::Aws(kid) = secret {
+                            let pk = kms.public_key(kid).await?;
+                            break '_block self.compute_key_id_for_public_key(&pk).await?;
```

Comment:
What does this syntax mean? It will step out of the `SecretType::NistP256` match, but when? After `self.compute_key_id_for_public_key(&pk).await?;` is done? only if it fails?

Reply:
Ok I got it now, it breaks to avoid continuing with the below `cfg_if!` code block right?